apdx-validation.tex

```diff
@@ -149,28 +150,35 @@ \subsubsection*{Rules for the \class{Identified} class}
 Reference: \sec{sec:Identified}}
 }
 
+\twotwozero{
+\printValid{The \sbol{wasGeneratedBys} property of an \sbol{Identified} object is OPTIONAL and MAY contain a set of \sbol{URI}s.  \\ Reference: \sec{sec:provenance}}
+}
+
+\twotwozero{
+\printComplete{Each \sbol{URI} contained by the \sbol{wasGeneratedBys} property of an \sbol{Identified} object MUST refer to an \sbol{Activity} object.\\ Reference: \sec{sec:provenance}}
```

Comment:
"wasGeneratedBys" : I believe that this should be singular, even thought that is against the typical SBOL convention, since the PROV-O property is named "prov:wasGeneratedBy"

This applies to all of the instances of singular vs. plural.

Reply:
I think our convention is fine to call this plural.  In the UML, we do not use the tag name.  For example, "sequences" is the list and "sequence" is the tag in a ComponentDefinition.  So, I think we are being consistent even though the tag in PROV-O is singular.

Reply:
OK.  If you don't think this will confuse people, I can be OK with it.

Reply:
These validation rules apply only at the level of library implementation, and do not affect serialization, correct?  The property is still serialized as `prov:wasGeneratedBy`?

I'm OK with writing the spec this way to be consistent, but practically speaking, I won't be in any great hurry to actually  implement this change in the libSBOL API.  Especially when there are so many other exciting tasks I could be working on. ;)

Reply:
The UML diagrams describe the API and not the actually serialization tags.  That is described separately via the examples.  For example, sbol's Name field is serialized as dcterms:title.  Also, the sequences field for ComponentDefinition is serialized as "sequence" for each individual sequence in the list.  We have used plural throughout SBOL in the API, so I just wanted to be convenient with it throughout.

Reply:
I agree with @jakebeal to the point that I'm voting no. The pluralization should be property -> properties, not an "s" on the predicate name.